Week5/Assignment5/app/src/main/java/com/raywenderlich/bootcamp/blockbusterrecyclerview/model/MoviesDataSource.kt

```diff
@@ -0,0 +1,61 @@
+package com.raywenderlich.bootcamp.blockbusterrecyclerview.model
+
+import com.raywenderlich.bootcamp.blockbusterrecyclerview.R
+
+class MoviesDataSource {
+
+    companion object {
+        fun getMoviesList(): ArrayList<Movie>  = arrayListOf(
```

Comment:
Nice work, ideally this would also be stored to the database as well, but you've utilized the database to store the favorites which meets the requirement of using room.